Dockerfile.debian

```diff
@@ -0,0 +1,37 @@
+ARG BASE_IMAGE
+
+FROM debian:${BASE_IMAGE}
+
+ARG PRINCE_VERSION
+ARG DEBIAN_VERSION
+ARG DEB_FILE=prince_${PRINCE_VERSION}-1_debian${DEBIAN_VERSION}_amd64.deb
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+      curl \
+      ca-certificates \
+      fonts-tlwg-garuda-otf \
+      fonts-lohit-orya \
+      fonts-lohit-mlym \
+      fonts-lohit-knda \
+      fonts-lohit-telu \
+      fonts-lohit-taml \
+      fonts-lohit-gujr \
+      fonts-lohit-guru \
+      fonts-lohit-beng-bengali \
+      fonts-lohit-deva \
+      fonts-baekmuk \
+      fonts-ipafont-mincho \
+      fonts-arphic-uming \
+      fonts-opensymbol \
+      fonts-liberation2 && \
+    curl --proto '=https' --tlsv1.2 -O https://www.princexml.com/download/${DEB_FILE} && \
```

Comment:
Are the proto and tlsv1.2 arguments necessary for security?

Reply:
It makes `curl` use https and and TLS1.2 or newer. This helps ensure that it actually uses https and doesn't fall back to an insecure TLS version — failing to do either of these things would suggest something is a bit fishy and warrants further investigation.